examples/plot_grid_search_census.py

```diff
@@ -163,7 +160,7 @@
 predictors = sweep.predictors_
 
 # %%
-# We could load these predictors into the Fairness dashboard now.
+# We could plot performance and fairness metrics of these predictors now.
 # However, the plot would be somewhat confusing due to their number.
```

Comment:
Their number -> the number of models?

Reply:
Yes, let me update that. I believe this same text exists in several places, too.

Reply:
Turns out it was just this one place, and I updated it!